### DISCUSSION TITLE
No separate node in structure viewer for documents from root document

### DISCUSSION THREAD

**Discussion by kuschuermann** [texstudio-org/texstudio#3050] [Ideas]
I might not understand all the thoughts that went into the current structure view, but I find it unusual and not particular helpful when I open a document shown as part of a root document and this creates a new and separate node for that very same document.

So now I have two entries in the structure viewer for the same document, the one under the root document and the node (with TODO and maybe other stuff) linked under it.

I feel that a more intuitive implementation would be to highlight (bold) the document listed under the root document and place TODO (and other?) entries directly under that node in the structure view.

I'd welcome insights one way or another, and would like to extend my gratitude for this really, really slick tool. I'm never going back to editing LaTeX in a plain text editor! :)

**Comment by sunderme** [comment]
have you looked at global toc ?

**Reply by kuschuermann** [reply]
Ooh, the Global ToC is actually closer in functionality, thank you (I had to install an AppImage to see that, as Debian 11 seems stuck on v3.0.4).

Is there a way to change the bright green background on Global ToC entries?

**Comment by octaeder** [comment]
you can change this in the config here:
![image](https://user-images.githubusercontent.com/102688820/229248263-7eb9cd99-48a9-49de-9b4c-cd83974fe553.png)

**Reply by kuschuermann** [reply]
Oh nice, I had missed the Advanced check option. Thank you!
